src/QueryPipeline/printPipeline.h

```diff
@@ -17,9 +17,11 @@ void printPipeline(const Processors & processors, const Statuses & statuses, Wri
     out << "  rankdir=\"LR\";\n";
     out << "  { node [shape = rect]\n";
 
-    auto get_proc_id = [](const IProcessor & proc) -> UInt64
+    auto get_proc_id = [](const IProcessor& proc) -> std::size_t
     {
-        return reinterpret_cast<std::uintptr_t>(&proc);
+        static std::unordered_map<const void*, std::size_t> pointer_to_id;
```

Comment:
static member is not a good choice because this map will be in memory until process finishes. better to allocate map outside of lambda and pass as an argument.